packages/machine-extractor/src/actions.ts

```diff
@@ -213,7 +259,75 @@ export const AssignAction = wrapParserResult(
     return {
       node: result.node,
       action: assign(result.argument1Result?.value || defaultAction),
-      name: '',
+      name: 'assign',
```

Comment:
thinking out loud: we could also hoist `.kind` to this return value here - so we could leverage exhaustive switches when consuming all `.kind: 'builtin'`+`.name` in `getActionConfig`. Right now we rely on a `default:` fallback that extracts unmatched things as inline

Reply:
Good idea. Done.